### PR TITLE
update to 0.7 with fixed timeframe

### LINKED ISSUE #10: push data sequence to short
## Sequence length

If you look at line 27 you see that

```
uint64_t TIME_TO_SLEEP = 60;
```

That is just one minute, not every two minutes (or 120 seconds). This needs to be fixed!

### PR DESCRIPTION
**Pull request by kreier**
Please check for other changes.

### CODE REVIEW

File: software/SolarAISVN/SolarAISVN.ino [hunk 1]
```diff
@@ -1,8 +1,13 @@
...
+// Solar- and windmeter at AISVN v0.7 
+// 2020/06/15
+//
+//
+// voltages converted on ESP32
+// submission time back to 120 seconds
+//
+// pin:       32,      33,       34,       35,   14,   26,   27,     12,   13
+// value:  solar, battery, currentA, currentB, load, wind, dump, solar2, LiPo
+// submit: solar, battery, current, load, wind, solar2, LiPo, bootCount
```
Comment (aisvn-data) [inline software/SolarAISVN/SolarAISVN.ino:10]: Extended documentation in the header is helpful

File: software/SolarAISVN/SolarAISVN.ino [hunk 2]
```diff
@@ -24,15 +29,14 @@ const char* server = "maker.ifttt.com";
 // Time to sleep
 uint64_t uS_TO_S_FACTOR = 1000000;  // Conversion factor for micro seconds to seconds
 // sleep for 2 minutes = 120 seconds
-uint64_t TIME_TO_SLEEP = 60;
+uint64_t TIME_TO_SLEEP = 120;
```
Comment (aisvn-data) [inline software/SolarAISVN/SolarAISVN.ino:32]: this fixes the reported time submission bug #11

File: software/SolarAISVN/SolarAISVN.ino [hunk 3]
```diff
@@ -112,8 +116,8 @@ void makeIFTTTRequest() {
   Serial.println(resource);
 
   String jsonObject = String("{\"value1\":\"") + voltage[0] + "|||" + voltage[1] + "|||" + voltage[2]
-                          + "\",\"value2\":\"" + voltage[3] + "|||" + voltage[4] + "|||" + voltage[5]
-                          + "\",\"value3\":\"" + voltage[6] + "|||" + voltage[7] + "|||" + voltage[8]
+                          + "\",\"value2\":\"" + voltage[4] + "|||" + voltage[5]
+                          + "\",\"value3\":\"" + voltage[7] + "|||" + voltage[8]
```
Comment (aisvn-data) [inline software/SolarAISVN/SolarAISVN.ino:120]: What happened to the submission of the third parameter?

File: software/SolarAISVN/SolarAISVN.ino [hunk 4]
```diff
@@ -159,18 +162,14 @@ void measureVoltages() {
...
   voltage[6] = int((voltage[6]) * 1.608 + 258);         // pin27 dump     not connected yet
   voltage[7] = int((voltage[7]) * 4.583 + 735);         // pin12 Solar2   voltage divider 4.7k : 1k
   voltage[8] = int((voltage[8]) * 1.608 + 258);         // pin13 LiPo     voltage divider 100kOhm 2:1
   Serial.print("Boot number: ");
   Serial.println(bootCount);  
-  Serial.print("Millisecond to measure: ");
-  //zeit = zeit - millis();
-  Serial.print(millis());
-  Serial.print(" - ");
-  Serial.println(zeit);
```
Comment (aisvn-data) [inline software/SolarAISVN/SolarAISVN.ino:175]: Old code from 0.6 to display the submission time on the console in different WiFi situations, no longer needed.

**Review by aisvn-data — COMMENTED** [review]
Please check the missing submission of voltages[3]

File: software/SolarAISVN/SolarAISVN.ino (hunk 3 above)
Comment (kreier) [reply software/SolarAISVN/SolarAISVN.ino:120]: look at line 10. The current A and current B are measured, but we don't need the voltage values, we need the raw current. Which would just be current - and this one is submitted.

**Pull request merged**